### PR TITLE
Use latest version of apktool

### PR DESCRIPTION
**Pull request by ritiek**
Add `apktool_2.2.4.jar` and made some modification to the script to use it. Ubuntu repositories provide an older version of apktool. This newer version could helpful in dealing with edge cases.

### CODE REVIEW

File: addSecurityExceptions.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ new="_new.apk"
 newFileName=$filename$new
 tmpDir=/tmp/$filename
 
-apktool d -f -o $tmpDir $fullfile
+java -jar apktool_2.2.4.jar d -f -o $tmpDir $fullfile
```
Comment (levyitay) [inline addSecurityExceptions.sh]: change to apktool.jar (no need for version, would make jar updates easier)

File: addSecurityExceptions.sh [hunk 2]
```diff
@@ -43,8 +43,8 @@ if ! grep -q "networkSecurityConfig" $tmpDir/AndroidManifest.xml; then
 fi
 
 
-apktool empty-framework-dir --force $tmpDir
+java -jar apktool_2.2.4.jar empty-framework-dir --force $tmpDir
```
Comment (levyitay) [inline addSecurityExceptions.sh]: change to apktool.jar (no need for version, would make jar updates easier)

File: addSecurityExceptions.sh [hunk 3]
```diff
@@ -43,8 +43,8 @@ if ! grep -q "networkSecurityConfig" $tmpDir/AndroidManifest.xml; then
...
 echo "Building new APK $newFileName"
-apktool b -o ./$newFileName $tmpDir
+java -jar apktool_2.2.4.jar b -o ./$newFileName $tmpDir
```
Comment (levyitay) [inline addSecurityExceptions.sh]: change to apktool.jar (no need for version, would make jar updates easier)

**Review by levyitay — COMMENTED** [review]
change to apktool.jar (no need for version, would make jar updates easier)

**Comment by ritiek** [comment]
Done. :smile:

I think we should also remove *Prerequisites* from README now.

**Pull request merged**